salt/states/boto3_efs.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python
```

Comment:
can you remove the hash bang?

Thanks.

Reply:
Removed hashbang